packages/@okta/vuepress-site/docs/guides/sampleapp-oie-redirectauth/main/index.md

```diff
@@ -128,37 +128,53 @@ You can modify the Application's Sign-On Policy to require the user to have a se
 
 2. From the side navigation, select **Applications** > **Applications** and then select the Okta OAuth app that you created to represent the <StackSnippet snippet="applang" inline /> app.
 3. Select the **Sign On** tab.
-4. Scroll down to the **Sign On Policy** section, click the **Actions** menu icon (&#8942;) beside the **ENABLED** flag and select **Edit**.
-5. In the Edit Rule dialog box, scroll down to the **THEN** section and locate **AND User must authenticate with**.
-6. Select **Password + Another factor** and click **Save**.
+4. Scroll down to the **User authentication** section and click **Edit**.
+5. From the **Authentication policy** dropdown menu, select **Any two factors** and click **Save**.
 
 ### Try multifactor authentication
 
 <StackSnippet snippet="tryenrollin" />
 
-3. Enter the credentials of the user that you enrolled earlier. The Set up authentications page appears, which prompts you to set up either the Okta Verify or the Phone authenticator.
+3. Enter the credentials of the user that you enrolled earlier. The Set up security methods page appears, which prompts you to set up either the Okta Verify app or the Phone authenticator.
 4. Under **Phone**, click **Set up**.
-5. Fill out the requested phone authentication information, verify your phone with a code, and then click **Finish**. You are redirected to the <StackSnippet snippet="applang" inline /> welcome page.
-6. Click <StackSnippet snippet="tryenrollout" inline /> to sign out of the <StackSnippet snippet="applang" inline /> app.
+5. Fill out the requested phone authentication information and verify your phone with a code.
+6. Under **Set up optional**, click **Set up later**. You are redirected to the <StackSnippet snippet="applang" inline /> welcome page.
+7. Click <StackSnippet snippet="tryenrollout" inline /> to sign out of the <StackSnippet snippet="applang" inline /> app.
+
+## Self-service Password Recovery
+
+### Email magic link password recovery flow
 
-## Authenticator recovery
+> **Note:** In your org, password reset is configured by default to be initiated with an email. The steps in this section assume that you haven't changed that default configuration.
```

Comment:
I love this note for making note of a default setting. I'm going to steal it. :)
```suggestion
> **Note:** In your org, password reset is configured by default to be initiated with an email. The steps in this section assume that you haven't changed that default configuration.
```

Reply:
:)